tensorboard/components/vz_line_chart2/vz-line-chart2.ts

```diff
@@ -411,10 +411,10 @@ namespace vz_line_chart2 {
         // asynchronous, and values may have changed in between the call being
         // initiated and actually being run.
         var chart = new LineChart(
-          xComponentsCreationMethod,
+          normalXComponentsCreationMethod,
           this.yValueAccessor,
-          yScaleType,
-          colorScale,
+          this.yScaleType,
+          this.colorScale,
           this.$.tooltip,
           this.tooltipColumns,
```

Comment:
I might make sense to remove these from the constructor but definitely not a required change.

Reply:
Ack, will keep these constructor params for now.